pkg/exchange/mexc/exchange.go

```diff
@@ -0,0 +1,233 @@
+package mexc
+
+import (
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+
+	"errors"
+	"fmt"
+	"io"
+	"strconv"
+	"context"
+	"time"
+	"crypto/sha256"
+	"crypto/hmac"
+	"net/http"
+	"net/url"
+	"encoding/json"
+)
+
+const MX = "MX"
+var urlTemplate url.URL = url.URL{
+	Scheme: "https",
+	Host: "api.mexc.com",
+	Path: "/api/v3",
+	RawPath: "/api/v3",
+}
+
+var log = logrus.WithField("exchange", "mexc")
+
+type Exchange struct {
+	key, secret string
+	client *http.Client
+}
+
+func (e *Exchange) Name() types.ExchangeName {
+	return types.ExchangeMEXC
+}
+
+func (e *Exchange) PlatformFeeCurrency() string {
+	return MX
+}
+
+func (e *Exchange) publicRequest(ctx context.Context, method string, path string, params url.Values) ([]byte, error) {
```

Comment:
try requestgen, it could reduce a lot of efforts! check kucoin's code, you can simply copy the http client code and reuse it.